indigo/distribution.yaml

```diff
@@ -7587,6 +7587,21 @@ repositories:
       url: https://github.com/ros-drivers/prosilica_gige_sdk.git
       version: hydro-devel
     status: maintained
+  pugixml:
+    doc:
+      type: git
+      url: https://github.com/joselusl/pugixml.git
+      version: 1.7.1
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/joselusl/pugixml-release.git
+      version: 1.7.1-0
+    source:
+      type: git
+      url: https://github.com/joselusl/pugixml.git
+      version: 1.7.1
```

Comment:
Fixed! Thanks!
Before seen this comment, I tried to open other pull request because I though I had made a mistake in the url of the documentation.
I have already closed the other pull request.
Sorry for the inconveniences, it is my first time and I am still learning :S